src/views/view.c

```diff
@@ -1068,6 +1069,39 @@ int dt_view_image_get_surface(int imgid, int width, int height, cairo_surface_t
     cairo_destroy(cr);
   }
 
+  // hack: in tether mode, use mipmap as adequate data to create a histogram
```

Comment:
I just tried using export code in place of this. Results seem better, and there's no need to use noise to fix quantization error. I'll work on this a moment more and then add one more commit.